pulp_ansible/app/galaxy/v3/views.py

```diff
@@ -90,7 +90,9 @@ def get_queryset(self):
         """
         distro_content = self.get_distro_content(self.kwargs["path"])
 
-        collections = CollectionVersion.objects.select_related("collection").filter(
+        collections = CollectionVersion.objects.select_related(
+            "content_ptr__contentartifact"
```

Comment:
Maybe this change is good to make in general because we were running the query one-by-one on `retrieve` and now we're doing it once in the get_queryset which is likely better. Do we need to make this as part of this PR though? I was wondering because I see get_queryset being redefined once again on UnpaginatedCollectionVersionViewSet which I believe is a descendant of this mixin right?

Reply:
you are right, it doesn't necessarily need to be on this PR

Reply:
Want to break it out to another one? It could be a [noissue] I think.

Reply:
oh! wait!
actually, this change is needed, because it affects how we use the serializer